### PR TITLE
Restorant card component

### PR DESCRIPTION
**Pull request by Bora1710**
Create reusable component to display restaurant and move elements

### CODE REVIEW

File: src/app/restaurant/restaurant.component.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
-.restaurant, .container, .singleReview {
+.container,
+.singleReview {
```
Comment (m-jovan) [inline src/app/restaurant/restaurant.component.css]: Class names should be kebab case (e.g. `single-review`). Also there's no need for the 'single' numerization in the name, `review` is just fine and simpler.

File: src/app/restaurantcard/restaurantcard.component.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+import { Restaurant } from '../shared/Models/restaurant';
+
+@Component({
+  selector: 'app-restaurantcard',
```
Comment (m-jovan) [inline src/app/restaurantcard/restaurantcard.component.ts]: Restaurant card is two words so they should be separated with a dash. `app-restaurant-card`

File: src/app/restaurantcard/restaurantcard.component.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  templateUrl: './restaurantcard.component.html',
+  styleUrls: ['./restaurantcard.component.css'],
+})
+export class RestaurantcardComponent {
```
Comment (m-jovan) [inline src/app/restaurantcard/restaurantcard.component.ts]:
```suggestion
export class RestaurantCardComponent {
```

File: src/app/restaurantcard/restaurantcard.component.ts (hunk 2 above)
Comment (m-jovan) [reply src/app/restaurantcard/restaurantcard.component.ts]: This also goes for the file and folder name of the component

File: src/app/restaurant/restaurant.component.html [hunk 4]
```diff
@@ -1,18 +1,5 @@
...
-
-  <ng-container *ngFor="let star of stars">
-    <app-star-icon
-      [ngClass]="{ active: star <= restaurant.averageRating! }"
-    ></app-star-icon>
-  </ng-container>
-
+<app-restaurantcard [restaurant]="restaurant"></app-restaurantcard>
+<div class="container">
   <form [formGroup]="reviewForm" (ngSubmit)="onSubmit()">
```
Comment (m-jovan) [inline src/app/restaurant/restaurant.component.html:3]: Is review form going to be a separate component as well?

Comment (Bora1710) [reply src/app/restaurant/restaurant.component.html:3]: It should be done in this branch?

Comment (m-jovan) [reply src/app/restaurant/restaurant.component.html:3]: No another one is ok, just checking

**Review by m-jovan — APPROVED** [review]

**Pull request merged**